docs/modifyingfileextensionsinwebform.md

```diff
@@ -16,48 +16,64 @@ When creating, for example, a `DigitalDocument` Object in Archipelago and you re
 
-![Manage > Structure > Webforms](../imgs/modifyingfileextensionsinwebform/03_manage-structure-webforms.jpg)
+![Manage > Structure > Webforms](../imgs/modifyingfileextensionsinwebform/02_manage-structure-webforms.jpg)
 
-This is where all of the Webforms inside your Archipelago live. In this screenshot you'll notice a few custom webforms that have been created to fit the needs of this particular repository: `Digital Object Collection Webform`, `My Favorite Webform` and `Student Webform`. For this guide we're going to edit the default Webform `Descriptive Metadata`. Go ahead and click `Build` under the `OPERATIONS` column for `Descriptive Metadata`.
+This is where all of the Webforms inside your Archipelago live. For this guide we're going to edit the Webform `Descriptive Metadata`, which is the default Webform shipped with Archipelago. Go ahead and click `Build` under the `OPERATIONS` column for `Descriptive Metadata`.
 
-![Descriptive Metadata](../imgs/modifyingfileextensionsinwebform/04_webform-page.jpg)
+![Descriptive Metadata](../imgs/modifyingfileextensionsinwebform/03_webform-page.jpg)
 
 ### Step 3: Editing Elements
 
-Here we see all of the elements in `Descriptive Metadata`; *Title*, *Media type*, *Description*, *Linked Data* elements, etc. The element that we want to edit is `Upload Associated Documents` as this is the field you will use to upload `pdf`, `doc`, `rtf`, `txt`, etc. files during the ingest workflow.  Go ahead and click on `Edit` under the `OPERATIONS` column.
+Here we see all of the elements in `Descriptive Metadata`; *Title*, *Media type*, *Description*, *Linked Data* elements, etc. The element that we want to edit is `Upload Associated Documents` as this is the field you will use to upload `pdf`, `doc`, `rtf`, `txt`, etc. files during the ingest workflow. Click on `Edit` under the `OPERATIONS` column.
 
-![Upload Associated Documents element](../imgs/modifyingfileextensionsinwebform/05_upload-associated-documents.jpg)
+![Upload Associated Documents element](../imgs/modifyingfileextensionsinwebform/04_upload-associated-documents.jpg)
 
-A new screen will pop up named `Edit Upload Associated Documents element`. This is where you configure can configure maximum number of values (under `ELEMENT SETTINGS`), the maximum file size and edit the allowed file extensions for this element, which is what we'll be doing. The latter both exist under `FILE SETTINGS`, high lighted in the screenshot below.
+A new screen will pop up named `Edit Upload Associated Documents element`. This is where you configure can configure the maximum number of values (under `ELEMENT SETTINGS`), the maximum file size and also edit the allowed file extensions for this element, which is what we'll be doing. The latter both exist under `FILE SETTINGS`, high lighted in the screenshot below.
```

Comment:
`This is where you configure can configure ` is double configure, looks like a type